src/components/shared/Accordion.css

```diff
@@ -15,18 +15,22 @@
 .accordion {
   background-color: var(--theme-sidebar-background);
   width: 100%;
+  list-style-type: none;
+  padding-left: 0px;
+  margin-top: 0px;
 }
 
 .accordion ._header {
   background-color: var(--accordion-header-background);
```

Comment:
Since this is now a `h2`, we should add `font-weight: normal` to prevent the bolding.